cuda_bindings/cuda/bindings/_internal/nvjitlink_linux.pyx

```diff
@@ -41,10 +41,10 @@ cdef int get_cuda_version():
         raise NotSupportedError(f'CUDA driver is not found ({err_msg.decode()})')
     cuDriverGetVersion = dlsym(handle, "cuDriverGetVersion")
     if cuDriverGetVersion == NULL:
-        raise RuntimeError('something went wrong')
+        raise RuntimeError('Did not find cuDriverGetVersion symbol in libcuda.so.1')
```

Comment:
same here

Reply:
This is moot if we nuke the entire function. See the linked issue and my comment below.